oceanmesh/edgefx.py

```diff
@@ -200,3 +216,75 @@ def wavelength_sizing_function(
         grid.values[grid.values > max_edge_length] = max_edge_length
     grid.build_interpolant()
     return grid
+
+
+def multiscale_sizing_function(
```

Comment:
blend_width should probably be in meters as it depends on the finer grid's resolution.